lib/ui/discover/StandardListItem.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+
+import 'discover_page.dart';
+
+
+abstract class ArticleListItemStyle {
+  final TextStyle titleTextStyle;
+  final TextStyle summaryTextStyle;
+
+  final EdgeInsets listEdgePadding;
+  final EdgeInsets cardMargin;
+
+  final double imageLineSpace;
+  final double textLineSpace;
+  final double imageSize;
+  final double cardElevation;
+
+  final BorderRadius imageBorderRadius;
+
+  final int maxLinesPerTitle;
+  final int maxLinesPerSummary;
+
+  ArticleListItemStyle(this.titleTextStyle, this.summaryTextStyle,
+      this.listEdgePadding, this.cardMargin, this.imageLineSpace,
+      this.textLineSpace, this.imageSize, this.cardElevation,
+      this.imageBorderRadius, this.maxLinesPerTitle, this.maxLinesPerSummary);
+
+}
+
+class _DefaultArticleListItemStyle implements ArticleListItemStyle {
+  static const Color titleColor = Color(0xFF1a1b46);
+  static const Color bodyColor = Color(0xFF767690);
+
+  final TextStyle titleTextStyle = const TextStyle(fontSize: 17, fontWeight: FontWeight.w700, color: titleColor);
+  final TextStyle summaryTextStyle = const TextStyle(fontSize: 15, fontWeight: FontWeight.w400, color: bodyColor);
+
+  final EdgeInsets listEdgePadding = const EdgeInsets.only(left: 32.0, right: 32.0, top: 23, bottom: 23);
+  final EdgeInsets cardMargin = const EdgeInsets.all(0);
+
+
+  final double imageLineSpace = 30.5;
+  final double imageSize = 80;
+  final double textLineSpace = 9.5;
+  final double cardElevation = 0;
+
+  final BorderRadius imageBorderRadius = const BorderRadius.all(Radius.circular(10.0));
+
+  final int maxLinesPerTitle = 2;
+  final int maxLinesPerSummary = 2;
+
+  const _DefaultArticleListItemStyle();
+}
+
+class HeroArticleListItemStyle implements ArticleListItemStyle {
```

Comment:
the hero style in the StandardListItem.dart? doesn't make sense. Would be better to lift this into the Hero widget file and use it as the private _DefaultListItemStyle.